_posts/2021-03-03-cockpit-239.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Cockpit 239
+author: lis
+date: 2021-03-03
+tags: cockpit linux
+slug: cockpit-239
+category: release
+summary: Cockpit with restyled updates page
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 239.
+
+### Machines: Allow editing disk cache mode
+
+It's now possible to edit the cache mode for existing virtual machine
+images.  This was previously only possible when creating new images.  In
```

Comment:
s/images/disks

Reply:
I'd still include the screenshot from https://github.com/cockpit-project/cockpit/pull/15430: 

![screenshot](https://user-images.githubusercontent.com/42733240/109371195-22dfe400-78a4-11eb-8a3c-5bc82ca82e84.png)

That is well done and makes the blog post less of a wall of text.

Reply:
Wow. Why is the storage volume name so high up? I guess it's in a tag that makes it monospaced, but it's still _so_ wrong.

Reply:
Filed @ https://github.com/cockpit-project/cockpit/issues/15459